packages/prisma/seed-app-store.ts

```diff
@@ -293,7 +294,17 @@ export default async function main() {
   );
   for (let i = 0; i < generatedApps.length; i++) {
     const generatedApp = generatedApps[i];
-    await createApp(generatedApp.slug, generatedApp.dirName, generatedApp.categories, generatedApp.type);
+    if (generatedApp.isTemplate && process.argv[2] !== "seed-templates") {
```

Comment:
It ensures that templates are seeded only if `seed-templates` argument is provided, which prevents templates from ever being seeded into production